Servers/using-runner-to-configure-disks-on-bare-metal-servers.md

```diff
@@ -0,0 +1,114 @@
+
+{{{
+"title": "Using Runner to configure disks on Bare Metal servers",
+"date": "10-17-2016",
+"author": "Anthony Hakim",
+"attachments": [],
+"contentIsHTML": false,
+"sticky": false
+}}}
+
+### Overview
+
+Customers who build particular Bare Metal server types (see below for prerequisites), have the ability to customize the configuration using both JBOD (HBA) and RAID modes. For those customers who would like an automated process to configure all disks on your Bare Metal server into one RAID set or JBOD, please follow the instructions below.
+
+If you'd prefer to carve the disks up in more of a custom fashion, such as multiple RAID sets or something more complicated, you can follow the step-by-step instructions in the [Configure the RAID controller on Bare Metal servers](../Servers/configuring-raid-on-bare-metal-servers.md) KB article.
+
+### Prerequisites

```

Comment:
with images/bare_metal_runner_disks_1.png removed from the article probably should remove the image from the pull request